### PR TITLE
Fills-in implied setup steps with explicit instructions

### PR DESCRIPTION
**Pull request by base10**
* Adds info on accounts required
* Adds info on connecting dev instance to a Sandbox
* For: https://trello.com/c/IxQLCcG5/28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,31 +11,76 @@ Planned integrations:
...
 
 ```sh
 docker-compose up
 ```
 
-3. Run the tests:
+### 5. Set-up the database
+
+```sh
+docker-compose run rake db:create
```
Comment (jferris) [inline README.md]: You can combine these two tasks using `rake db:setup`.

File: README.md [hunk 2]
```diff
@@ -11,31 +11,76 @@ Planned integrations:
...
 ```sh
 docker-compose run web rake
 ```
 
+### 7. Required accounts
+
+Make sure you have accounts or access for the following:
+
+* Heroku Staging
+* Cloud Elements
```
Comment (jferris) [inline README.md]: There are a number of other accounts involved, such as Jobvite, but you only need accounts or access if you're working on that particular integration. Developers who are adding new integrations won't need NetSuite accounts. They'll only need Cloud Elements if they're adding another integration through Cloud Elements.

**Comment by base10** [comment]
@jferris NRR

**Comment by jferris** [comment]
Looks good.

**Comment by base10** [comment]
43a342c is where the merge happened. Deleted the branch ahead of pushing the merge.

**Pull request closed without merging**